src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -26,24 +27,29 @@ import nl.knaw.dans.easy.fedora2vault.Command.FeedBackMessage
 import nl.knaw.dans.easy.fedora2vault.FoXml.{ getEmd, _ }
 import nl.knaw.dans.lib.error._
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import nl.knaw.dans.pf.language.emd.EasyMetadataImpl
+import nl.knaw.dans.pf.language.emd.binding.EmdUnmarshaller
+import org.joda.time.DateTime
 
+import scala.collection.JavaConverters._
 import scala.util.{ Failure, Success, Try }
-import scala.xml.{ Elem, Node, XML }
+import scala.xml.{ Elem, Node }
 
 class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLogging {
-  lazy val fedoraProvider: FedoraProvider = new FedoraProvider(new FedoraClient(configuration.fedoraCredentials))
+  lazy implicit val fedoraProvider: FedoraProvider = new FedoraProvider(new FedoraClient(configuration.fedoraCredentials))
```

Comment:
Is this `implicit` necessary here? I mean, I see a requirement for `implicit` here (https://github.com/DANS-KNAW/easy-fedora2vault/pull/2/files#diff-323ba07a551f97fa3f0e08f972fc3741R122-R126), but that's in the same file.